core/src/avm2/globals/flash/display/InteractiveObject.as

```diff
@@ -1,13 +1,25 @@
 package flash.display {
+    import __ruffle__.stub_setter;
 
+    import flash.accessibility.AccessibilityImplementation;
 	import flash.ui.ContextMenu;
 
 	[Ruffle(NativeInstanceInit)]
 	public class InteractiveObject extends DisplayObject {
+	    private var _accessibilityImpl:AccessibilityImplementation = null;
+
 		public function InteractiveObject() {
 			throw new Error("You cannot directly construct InteractiveObject.")
 		}
 
+		public function get accessibilityImplementation():AccessibilityImplementation {
+		    return this._accessibilityImpl;
+		}
+		public function set accessibilityImplementation(value:AccessibilityImplementation):void {
+		    stub_setter("flash.display.InteractiveObject", "accessibilityImplementation");
```

Comment:
Is this a stub if it's actually doing something?

Reply:
No, this isn't doing anything. Calling the setter should update the accessibility implementation (unlike `DisplayObject.accessibilityProperties`, which you must call `Accessibility.updateProperties()` to update)- this setter doesn't do anything right now.